src/jekyll/_plugins/custom_tags/include_code.rb

```diff
@@ -151,18 +151,18 @@ def render_codehighlighter(context, code, filepath)
 
       link_text = site.data["localized_strings"][page["langcode"]]["try_sample"]
       link_text = link_text ? link_text : site.data["localized_strings"]["en"]["try_sample"]
+      #TODO: Move the HTML snippet below into a liquid component so it can
```

Comment:
I thought about that, but worried that they'd potentially get lost as files moved or were updated. By putting them inline, when this code is updated the comment is front of mind. Happy to file issues in addition to the TODO comment, if you feel it's helpful.